documentation/Installation.md

```diff
@@ -3913,8 +3913,8 @@ The plugin configuration supports the following parameters:
 
 | Name                   | Type    | Default Value                       | Value Rules                                      | Description                                                        |
 |------------------------|---------|-------------------------------------|--------------------------------------------------|--------------------------------------------------------------------|
-| mode                   | string  | `ipip`                              | `ipip` / `vxlan`                                 | Network protocol to be used in network plugin                      |
-| crossSubnet            | boolean | `true`                              | true/false                                       | Enables crossing subnet boundaries to improve network performance  |
+| mode                   | string  | `ipip`                              | `ipip` / `vxlan` / `none`                        | Network overlay technology to be used in network plugin                      |
+| crossSubnet            | boolean | `true`                              | true/false                                       | Enables crossing subnet boundaries to avoid additional network configuration |
```

Comment:
This parameter is used in conjunction with `plugin.calico.mode` to define when the encapsulation (ipip/vxlan) should be turned on:
https://github.com/Netcracker/KubeMarine/blob/5cf0b6da975807bba1684b3f888fc9412558b66f/kubemarine/templates/plugins/calico-ippool.yaml.j2#L10-L11
Probably it should be described more explicitly, what its true or false value would mean.

Reply:
Updated a bit